### DISCUSSION TITLE
Getting prisma to play with aws lambda serverless typescript using layers

### DISCUSSION THREAD

**Discussion by kaykhan** [prisma/prisma#15222] [Q&A] [answered]
I have a repostiory im working on, we have created a `shared` folder which contains shared functions which ought to be re-used by other apps in this repository. This shared folder also holds our prisma schema and reusable functions to interface with the database. 

Following this guide we attempt to use a prisma layer . - https://dev.to/sabinthedev/managing-lambda-size-with-multiple-prisma-clients-4m8l

However we have found that this does not work for us, giving us the following error:

```
:heavy_multiplication_x: Error: ENOENT: no such file or directory, open '/home/kay/kay/prisma-example-typescript/app1/.esbuild/.build/schema.prisma'
```

I have created a minimal reproduction repo here - https://github.com/kaykhan/prisma-typescript-example - the serverless function (app1) was bootstraped using `sls create --template aws-nodejs-typescript --path app1`

**Im looking for an answer that would get this to work using prisma layers, like the guide linked above.**

**Comment by austincrim** [comment, marked as answer]
It looks like this question was already discussed and handled in [this Slack thread](https://prisma.slack.com/archives/CA491RJH0/p1662391155595529). 

Feel free to open a new discussion if you have other questions!

**Reply by elitan** [reply]
Can you share the solution here on GitHub, so it's open and public to anyone?

**Reply by GermaVinsmoke** [reply]
@elitan  Doesn't look like they got an answer to this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
